src/koa_rti_main.py

```diff
@@ -12,7 +13,7 @@
 
 from ingest_api.ingest_api import ingest_api_get
 from utils.koa_rti_api import KoaRtiApi
-from utils.koa_rti_helpers import get_api_help_string, InstrumentReport
+from utils.koa_rti_helpers import get_api_help_string#, InstrumentReport
```

Comment:
I suggest removing the comment, or describing why it is commented out. As is it looks unfinished.